docs/content/about-iota/iota-architecture/epochs.mdx

```diff
@@ -9,42 +9,49 @@ import {questions} from '../../../site/static/json/about-iota/iota-architecture/
 
 ## Epoch
 
-In IOTA, an epoch is a set period, around 24 hours, during which the
-IOTA [validator](../tokenomics/validators-staking.mdx) set and their stakes remain constant. This stable period allows
-validators to handle transactions smoothly without unexpected changes.
+In IOTA, an epoch is a fixed period (typically around 24 hours) during which the [validator](../tokenomics/validators-staking.mdx) set and their stakes remain constant. This stable period ensures that [transactions](../../developer/iota-101/transactions/transactions.mdx) are processed smoothly without unexpected validator changes.
```

Comment:
Make a "validator set" refer to "consensus committee": `[validator set](./consensus.mdx#consensus-committee)`.